cluster/examples/workloads/kubernetes/wordpress/aws/network-config/dbsubnetgroup.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: storage.aws.crossplane.io/v1alpha2
```

Comment:
Pending change in https://github.com/crossplaneio/stack-aws/pull/48

Reply:
ack

Reply:
@hasheddan I updated this to `apiVersion: database.aws.crossplane.io/v1alpha2`